Makefile

```diff
@@ -114,38 +118,43 @@ test: manifests generate fmt vet test-unit test-e2e #HELP Run all tests.
 
 .PHONY: e2e
 e2e: $(SETUP_ENVTEST) #EXHELP Run the e2e tests.
-	go test -tags $(GO_BUILD_TAGS) -v ./test/e2e/...
+	go test -v ./test/e2e/...
 
-export REG_PKG_NAME=registry-operator
-export PLAIN_PKG_NAME=plain-operator
-export CATALOG_IMG=${E2E_REGISTRY_NAME}.${E2E_REGISTRY_NAMESPACE}.svc:5000/test-catalog:e2e
+E2E_REGISTRY_NAME      := docker-registry
+E2E_REGISTRY_NAMESPACE := operator-controller-e2e
+export REG_PKG_NAME   := registry-operator
```

Comment:
@joelanford I found a misalignment here ;) 

I would prefer if we could avoid formatting like this as it is hard to maintain this style. This also leads to larger diffs: for example, [here](https://github.com/operator-framework/operator-controller/pull/798/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52) I wanted to only remove one line but I had to change another for it to not look weird.

Reply:
https://github.com/operator-framework/operator-controller/pull/827